public/index.html

```diff
@@ -106,6 +108,69 @@ <h6>
       </footer>
     </div>
 
-    <script src="/reload/reload.js"></script>
+    <!-- Load Skeleton/Google FONT -->
+    <link
+      href="//fonts.googleapis.com/css?family=Raleway:400,300,600&display=swap"
+      rel="stylesheet"
+      type="text/css"
+      media="none"
+      media="none"
+      onload="this.media='all'"
+    />
+
+    <!-- Load CSS files -->
+    <link
+      href="css/normalize.min.css"
+      rel="stylesheet"
+      media="none"
+      media="none"
+      onload="this.media='all'"
+    />
+    <link
+      href="css/skeleton.min.css"
+      rel="stylesheet"
+      media="none"
+      media="none"
+      onload="this.media='all'"
+    />
+    <link
+      href="css/style.css"
+      rel="stylesheet"
+      media="none"
+      media="none"
+      onload="this.media='all'"
+    />
+
+    <!-- Load global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-163501590-1"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+
+      gtag('config', 'UA-163501590-1');
+    </script>
+    
+    <!-- Load tf.js libraries !-->
+    <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-core"></script>
+    <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-converter"></script>
+
+    <!-- Load the pre-trained models !-->
+    <script src="https://cdn.jsdelivr.net/npm/@tensorflow-models/facemesh"></script>
+    <script src="https://cdn.jsdelivr.net/npm/@tensorflow-models/handpose"></script>
+
+    <!-- Load WASM backend for tf.js !-->
+    <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-backend-wasm/dist/tf-backend-wasm.js"></script>
+
+    <!-- Load three.js -->
+    <script src="https://cdn.jsdelivr.net/npm/three@0.106.2/build/three.min.js"></script>
+
+    <!-- Load scatter-gl.js -->
+    <script src="https://cdn.jsdelivr.net/npm/scatter-gl@0.0.1/lib/scatter-gl.min.js"></script>
+
+    <!-- Load main local JS libraries -->
+    <script type="module" src="js/main.js" defer></script>
```

Comment:
as discussed in the meeting, consider not deferring this since it kicks off tfjs runtime load